### DISCUSSION TITLE
Setting Commodity vs Industry in 1.2.3 Confirmation of output Requested

### DISCUSSION THREAD

**Discussion by ranastasiadis** [USEPA/supply-chain-factors#69] [Q&A] [answered]
Hello, 

In threads #[42](https://github.com/USEPA/supply-chain-factors/discussions/42) , #[55](https://github.com/USEPA/supply-chain-factors/discussions/55), and #[65](https://github.com/USEPA/supply-chain-factors/discussions/65) it is mentioned that the model is set to a commodity model as per line 10 of the yml configuration file and can be configured to be industry.

Prior to PR #[61](https://github.com/USEPA/supply-chain-factors/pull/61), the dev 1.2.3 branch was required to successfully generate these files. 

I have generated, post 1.2.3, files for industry factors, I was hoping to confirm that these factors have generated correctly after the new pull request and that there are no additional requirements other than the commodity -> industry argument. 

Could you please advise if these factors have generated for industry specs as intended for someone downloading the 1.2.3 and changing only the industry vs commodity specs?

### Relv Files.
 [SupplyChainGHGEmissionFactors_v1.2.3_NAICS_CO2e_USD2021.csv](https://github.com/USEPA/supply-chain-factors/files/13649567/SupplyChainGHGEmissionFactors_v1.2.3_NAICS_CO2e_USD2021.csv)
[SupplyChainGHGEmissionFactors_v1.2.3_NAICS_byGHG_USD2021.csv](https://github.com/USEPA/supply-chain-factors/files/13649572/SupplyChainGHGEmissionFactors_v1.2.3_NAICS_byGHG_USD2021.csv)

Thank you!

Ryan

**Comment by bl-young** [comment, marked as answer]
While I can't review your output files in detail and an EPA-reviewed industry dataset has not been published, the number of sectors looks correct and the magnitude of emissions factors looks fine.

> and that there are no additional requirements other than the commodity -> industry argument

This is correct - adjusting the model specs should be all you need to do.
